pyproject.toml

```diff
@@ -4,17 +4,20 @@ version = "0.1.0"
 description = "Extend JAX with xDSL."
 readme = "README.md"
 requires-python = ">=3.10"
-dependencies = ["jax>=0.6.2", "xdsl==0.53.0"]
-classifiers = ["Programming Language :: Python :: 3 :: Only", "Typing :: Typed"]
+dependencies = [
+    "jax>=0.6.2",
+    "xdsl==0.54.3",
```

Comment:
Instead of pinning to a specific version of xdsl, would a lower bound make more sense?